knime-scripting-templates/R/snippet-templates.txt

```diff
@@ -623,13 +625,13 @@ Alternatively, the message 'Error: Could not find atrribute: treatment' can be i
 parameters = c(<panellistbox label="Features of interest" items="$$$NUM_ATTRIBUTES$$$" visible-row-count="6" span="full"/>)
 
 # 2) calculate mean and covariance of the negative controls
-treatment = <combobox items="$$$DOMAIN('treatment')$$$" label="Negative control name"/>
+treatment = <combobox items="$$$DOMAIN('treatment')$$$" label="Reference population name"/>
 <hbox>
-<combobox var = "treatmentColumn" label = "Negative control column" items = "-USE SELECTED TREATMENTS-,$$$STR_ATTRIBUTES$$$"/>
-<textfield label = "Negative control:" var="negColtrolName" data-type="text" default-value= "" size="10"/>
+<combobox var = "treatmentColumn" label = "Referene population column" items = "-USE SELECTED TREATMENTS-,$$$STR_ATTRIBUTES$$$"/>
+<textfield label = "Reference population:" var="negColtrolName" data-type="text" default-value= "" size="10"/>
 </hbox>
 <gaprow height="1"/>
-
+<vector  var="p_thresh" label="Outlier p-value" size="1" vector-type="numeric" default-value="0.001"/>
```

Comment:
I noticed a very strange result in the behaviour of RGG. If this volue 0.001 is changed it calculated the threshold correcly, but upon re-opening the snippet node configuration, the default value is back. 
I did not trouble shoot to the end, but it looks like it has to do with the vector-type. Play around with these two line of code...

<vector  var="p_thresh" label="Outlier p-value" size="1" vector-type="numeric" default-value="0.001"/>

<vector label="number of digits before/after decimal point" var="digitVector" size="1" default-value="3" vector-type="character"/>

Reply:
You are right. The numeric vector does not store the settings for some reason. I created an [RGG-issue](https://github.com/knime-mpicbg/rgg/issues/11)
As a workaround I will replace it with a non-numeric vector input and its value will be converted to a number.